powerline/segments/vim/plugin/ctrlp.py

```diff
@@ -0,0 +1,102 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+try:
+	import vim
+except ImportError:
+	vim = object()
+
+from powerline.bindings.vim import create_ruby_dpowerline
+
+
```

Comment:
By the way, `initialized` global should be before the function it is being used in.